otel-integration/k8s-helm/e2e-test/testdata/values-e2e-test.yaml

```diff
@@ -11,10 +32,9 @@ opentelemetry-agent:
   config:
     exporters:
       otlp:
-        endpoint: 172.17.0.1:4317 # Use the default Docker host gateway IP
+        endpoint: ${env:HOSTENDPOINT}:4317
```

Comment:
I would advice against relying on shell env var expansion here. If the OTEL exporter code changes and somehow starts to validate this endpoint address before env var expansion happens, we will have issues. I've fixed a similar bug recently in our OTEL collector chart.

I think we can rely on `host.docker.internal`, which has been adopted by a few tools (podman, colima, etc) already as the de-facto standard hostname for this.

Reply:
changes made to insert via helm config param, works